### DISCUSSION TITLE
2.0.0 release ?

### DISCUSSION THREAD

**Discussion by johnatas-x** [mglaman/phpstan-drupal#811] [General]
Hi,

First of all, thank you so much for all the hard work over the past weeks to ensure compatibility with PHPStan 2.0!
Additionally, a big thanks for the removal of Drupal 9 support—it’s much appreciated.

The updates seem to be working well on the "main" branch.

Would it be possible (or is it already planned) to create a 2.0.0 release from the main branch including these changes?

Thanks !

**Comment by mglaman** [comment]
Let me cut a release! It's mostly prepared, I just haven't really tested it. I'm just going to go for it and I will chase after any fixes

**Comment by mglaman** [comment]
https://github.com/mglaman/phpstan-drupal/releases/tag/2.0.0 :) all set

**Comment by johnatas-x** [comment]
Thank you again!

For now, everything is working as expected on my project with the main branch. I’m now moving to the 2.0.0 tag, and I won’t hesitate to reach out if I encounter any issues.
